oldabe/money_out.py

```diff
@@ -76,6 +73,47 @@ def prepare_message(balances: dict):
     return "\r\n".join(line.strip() for line in message.split('\n')).strip()
 
 
+def read_outstanding_debt_amounts():
+    outstanding_debts = defaultdict(int)
+    with open(DEBTS_FILE) as f:
+        for row in csv.reader(f):
+            d = Debt(*row)
+            d.amount = Decimal(d.amount)
+            d.amount_paid = Decimal(d.amount_paid)
+            outstanding_amount = d.amount - d.amount_paid
+            outstanding_debts[d.email] += outstanding_amount
+    return outstanding_debts
+
+
+def prepare_debts_message(outstanding_debts: dict):
+    if not outstanding_debts:
+        return "There are no outstanding (unpayable) debts."
```

Comment:
ok, I see how having the parens would make it clear in the message that these two categories represent payable vs unpayable